public/i18n.js

```diff
@@ -1,84 +1,179 @@
+    "ftm-index-where-to-find-ppe": "[!!Ľőŏĸ įŉ ɏōűŗ ƃȁšȇmęŉţ, ȳőųŕ ėměŗğėńĉɏ ƙīŧ, ȳơūř ƅūĝ ŏųŧ ƃȁĝ, ɏơũŗ ŵōřƙƀĕŉċĥ ƒőř Ɲ95 måśĸŝ.Įƒ ŷŏũ ƒıņđ šūŕĝĩĉǻľ mȃšƙš, ŧħŏŝė ăřę ȃľšơ ĩŉ şĥŏřţ ŝūƥƿłȳ, āŋď īń mãŉŷ ƥļāčěş äŗĕ ƀęĩŋġ ūšęď įń ƥŀáčė ŏƒ Ŋ95ş. Áłŀ ơƒ ţĥȇşē măşķŝ ćáŋ şāvȅ ľįvěş ňōŵ ıƒ ȳŏų ģȇţ ŧĥȅm ıŋţŏ ţħĕ ĥȃƞďŝ ơƒ ħȅäĺŧħ ćǻŗě ŵōřĸęŕş.!!]",
+    "ftm-index-types-of-ppe": "[!!Ơťĥěř ŧƴƿȇš ơƒ ƤƤĔ łĩƙė đĩśƥơşãƅłė <span class=embold>ƃŏơţįȅş</span>, <span class=embold>şăƒęťƴ ğőĝģĺęş</span>, áňď <span class=embold>đĩśƥŏşäƃŀĕ ŝůĩťś</span> ȁŕę ąľšơ ĩņ śħơřť śūƥƿļŷ. Ĭƒ ƴōů ħâvę ŧħęŝȅ ţĥįńĝš őŗ ĸŉőŵ şōmēōŉę ŵĥō mıĝĥť (ĦVÅČ ıƞşŧăŀĺėřş? Ċơŋśţŕūčŧĩŏń ŵōŕƙȇřš?) ƿļėȁŝĕ ŝĥâřė ţħĭś şįŧē ŵīŧĥ ŧĥěm. Ļąŕĝĕ đŏňơřš ćâņ ŗĕĝĩŝţĕŗ ąţ <a href=\"https://getusppe.org/\">ĝěťűşƥƿė.ōřğ/ğįvȅ</a>.!!]",
+    "ftm-what-to-do-header": "[!!Ĥėřě'ŝ ŵĥâť ȳőů ćǻņ ďō:!!]",
+    "ftm-what-to-do-1": "[!!Łőơĸ āť ŏűř ŀĩşţ ơƒ ŝįťěŝ Ƃėľŏŵ. Ÿōű ćȃŉ ƒĩŀţęř Ƃȳ şţâťē ơř şůƿƥŀīęş ŋēęďȇđ.!!]",
+    "ftm-what-to-do-2": "[!!Ĭƒ ťĥȇŗę'ś à śīŧȅ ŉėāŗ ŷơů, ďŗőƥ ōƒƒ ƒŏŀłơŵıńġ ŧĥȇīř įņŝŧřŭćŧıōŋš.!!]",
+    "ftm-what-to-do-3": "[!!Ĩƒ ȳơŭ ȃŗē ďőńãťįņġ ŏƥēƞ ƥăćĸȃģĕš, ƿŀěâšė ƿŭŧ ťħēm ĩƞ à źĭƥŀōć Ƃãğ ŵĩţĥ à ŋŏŧȇ šąŷįňĝ ŵĥēţĥȅŗ ŧĥĕȳ ĥāvȅ Ƃęęƞ ūšěđ, ŵĥĕţħȅŕ āňɏőńě ıƞ ɏőũř ħōųśĕĥőļđ ħåš ƅĕȇŉ śįćƙ őŕ ťřȃvĕĺľȇđ ŏųŧśįđę ţĥĕ ćōŭńťřŷ, áńď ťĥȅ ȇxƥīŗãŧĩŏŋ ďǻţě įƒ ƙňơŵƞ.!!]",
+    "ftm-what-to-do-4": "[!!Įƒ ťĥěŕě'ŝ ňŏŧ ǻ śĭťȅ ƞēàŗ ȳŏŭ, vıŝıť <a href=\"https://getusppe.org/give\">ġėţųšƿƥē.ŏřĝ</a> ťő ľęť ţĥēm ĸŉőŵ ŵĥȃť ŷőŭ ħãvę ävàīłȁƅłē.!!]",
+    "ftm-add-donation-site": "[!!Ţō ȃđđ à ďőňåťįōń śįťę, ųƿđâŧė ĩƞƒőřmǻţįŏņ, ŏŕ ŗěǭųęśţ ŵė ťǻĸė đŏŵŋ ĩňƒơřmåŧĩŏŉ, ƿłėȃśȇ <a href=\"https://docs.google.com/forms/d/e/1FAIpQLSfgCpK5coPVFC6rJrE7ZhimiZuDoEaL6fo6gYqxsN_FIpJZhg/viewform\">ƒĩłļ īň ŧĥīş ƒőŕm</a>.!!]",
+    "ftm-donations-site-loading": "[!!Ĺōǻđıƞĝ đőňáţĩŏŉ śĭŧėś...!!]",
+    "ftm-filters-header": "[!!Ƒĩŀţȅŕŝ!!]",
+    "ftm-project-leads": "[!!Őųŕ ļĕăďŝ ářę:!!]",
+    "ftm-site-launch-info": "[!!Ŧħįŝ şĭťě ŵȁŝ ľâűŉčħĕđ ōņ <a href=\"https://twitter.com/arrkay/status/1240704610186092546\">Mąŗċĥ 19ŧĥ</a> ƅƴ <a href=\"https://www.linkedin.com/in/rachelpopkin/\">Řǻćĥȇľ Ƥơƥƙĭŋ</a> ǻńď ħàś ƅēěƞ ımƿŗơvȇď ƃɏ máņɏ ƒãŋťåšŧīċ vơĺűŉţȅȅřś. Mȃŋɏ őƒ ũš áŕȅ ŧȅčħ ŵơřƙėŗş ıń Śęàŧťłė. Ŵȅ ȃŗē ȁďvĭśěđ Ƃɏ á ġŕőųƥ ōƒ đŏčŧōŗŝ âňď ňűŕŝĕš ăŕőūńď ŧħĕ ĉŏųňŧŕɏ.!!]",
+    "ftm-anonymous-volunteers": "[!!āńđ őŧħēŗŝ ŵħō ǻŗȅ āŋơņŷmőůş!!]",
+    "ftm-ack-data-sources": "[!!Ťħĭš ŝįŧĕ ąŀšō īńćļŭďěş đàŧà ƒŗŏm <a href=\"https://tinyurl.com/hospitals-needing-ppe\">ħŧţƿš://ŧĭņɏũŕŀ.ĉōm/ĥőśƿıţǻłś-ňēěđıņĝ-ƥƿȇ</a>, å ƥřŏĴȅċţ ƀƴ:!!]",
+    "ftm-project-volunteers": "[!!Őūŗ vơľũŋŧȇęŕś ĭňċłŭďė:!!]"
+  }
+};
 
-$(function() {
-  $.i18n().load( locales );
+$(function () {
+  const searchParams = new URLSearchParams((new URL(window.location)).search);
+  const country = searchParams.get('country');
+
+  const defaultLocale = 'en';
+
+  $.i18n({ locale: (!!locales[country] && country) || defaultLocale }).load(locales);
```

Comment:
This looks like it will either use `defaultLocale` (en) or the `country` param. What we lose doing it this way is automatic browser locale detection. A better way might be something along the lines of:
```
const country = searchParams.get('country');
if(!!locales[country] && country) {
  $.i18n({ locale: country }).load(locales);
} else {
  $.i18n().load(locales);
}
```

It would also work to fetch the browser language here instead of letting the library do it, but I don't think that's quite as good: https://stackoverflow.com/questions/673905/best-way-to-determine-users-locale-within-browser

Reply:
yes could call, I tested and that works much better. fr-FR as a param would falls back to fr. Garbage as the param falls back to english